### ISSUE TITLE
Weird behaviour on horizontally moving platforms

### ISSUE THREAD

**Issue by parar020100** [MattJeanes/TARDIS#1069]
This is suggested to be happening due to vertical landing features

Reported by @catradorasustainsme in Discord:

Example:
https://www.youtube.com/watch?v=3XtSI3XwgKg

**Comment by AmyJeanes** [comment]
Yeah looks like the landing features, once on the ground they should generally be disabled

**Comment by AmyJeanes** [comment]
Fixed in https://github.com/AmyJeanes/TARDIS/commit/6364c907beb74486496addf715de97f69b255a9c
